pkg/flag/scan_flags.go

```diff
@@ -37,6 +37,15 @@ var (
 			types.SecretScanner,
 			types.LicenseScanner,
 		}),
+		ValueNormalize: func(s string) string {
```

Comment:
Can we add test that this function works as expected?

Reply:
Done
https://github.com/aquasecurity/trivy/pull/5558/commits/6a9343fbbd337d06f6aef5f52af4cf89d4f66e99